### ISSUE TITLE
Jobs completion seems to exceed 100%

### ISSUE THREAD

**Issue by NANDoffDataRecovery** [abueide/harry-plotter#54]
![image](https://user-images.githubusercontent.com/84880573/119746091-853a8780-be87-11eb-80ba-3484e5482712.png)

**Comment by abueide** [comment]
yeah currently i'm naively estimating percentage by taking the current running time divided by the average completion time for that plot. in the future I'll have a better estimation that takes into account its phase/subphase progress. I just thought this way would be easy & more useful than nothing

**Comment by NANDoffDataRecovery** [comment]
Fair enough :-)
